src/apb/dat/ex.apb.yml

```diff
@@ -1,10 +1,12 @@
-name: sample-apb
-image: organization/sample-apb
-description: "This is a sample skeleton file generated by `apb init`."
-bindable: false
-async: optional
+name: {{ apb_dict['apb-name'] }}
+image: {{ apb_dict['organization'] }}/{{ apb_dict['apb-name'] }}
+description: {{ apb_dict['description'] }}
+bindable: {{ apb_dict['bindable'] }}
+async: {{ apb_dict['async'] }}
 parameters:
-  - name: sample_param
-    description: Sample description of APB
-    type: string
-    default: sample-param-string
+{% for param in params -%}
```

Comment:
@jmrodri https://github.com/dymurray/ansible-playbook-bundle/blob/44a3202a1fc140fa1c2769c60169d6d62642b35b/src/apb/engine.py#L58 Is where params gets set if it was not passed in